test/orange/surf/Involute.test.cc

```diff
@@ -59,6 +59,8 @@ TEST(InvoluteTest, sense)
         {1, 0}, 1.0, 0.5 * pi, ccw, 1.732050808, 1.732050808 + 1.99 * pi};
     EXPECT_EQ(SignedSense::outside, invo.calc_sense({0, 0, 1}));
     EXPECT_EQ(SignedSense::outside, invo.calc_sense({0, 1.0, 1}));
+    EXPECT_EQ(SignedSense::outside, invo.calc_sense({2.999, 0, 1}));
+    EXPECT_EQ(SignedSense::inside, invo.calc_sense({3.001, 0, 1}));
```

Comment:
I reverted the involute test to the "buggy" version (where only the y component was being divided by the radius) and these tests still pass: a key thing during development is when you catch an error, you make sure you add a test that fails before and passes after, otherwise that error may someday creep back in. Maybe you should use python to sample a bunch of random points on a disk between an inner and outer radius for an involute, feed them into the unit test and output whether they're "inside" or "outside", and then plot the points in python (with the involute shape) and make sure the results are all sensible.